lib/pdk/validators/base_validator.rb

```diff
@@ -45,7 +45,7 @@ def self.parse_targets(options)
               skipped << target if target_list.flatten.empty?
               target_list
             elsif File.file?(target)
-              if target.eql? pattern
+              if Array(pattern).flatten.include? target
```

Comment:
No need to call `flatten` after `Array()` unless `pattern` is going to be a multidimensional array (`Array([1, 2, 3])` => `[1, 2, 3]`)